Sources/Commands/SwiftTestCommand.swift

```diff
@@ -656,6 +677,10 @@ extension SwiftTestCommand {
         @OptionGroup()
         var testLibraryOptions: TestLibraryOptions
 
+        /// Options for Swift Testing's event stream.
+        @OptionGroup()
+        var testEventStreamOptions: TestEventStreamOptions
```

Comment:
Is this really for just configuration rather than the stream options?

Reply:
I don't understand, sorry?

Reply:
This is in `list`. Does `list` actually need all these, or does it just need the configuration path?

Reply:
`list` can use them all now! See https://github.com/apple/swift-testing/issues/511